src/components/StudyCreate/StudyCreateMbtiSelect/StudyCreateMbtiSelect.tsx

```diff
@@ -0,0 +1,60 @@
+import Checkbox from '@mui/material/Checkbox';
+
+import MbtiTag from '../../MbtiTag/MbtiTag';
+
+import { MbtisWrapper } from './style';
+
+const mbtisList = [
+  'INFP',
+  'INFJ',
+  'INTP',
+  'INTJ',
+  'ISTJ',
+  'ISTP',
+  'ISFJ',
+  'ISFP',
+  'ENFP',
+  'ENTP',
+  'ENFJ',
+  'ENTJ',
+  'ESFP',
+  'ESFJ',
+  'ESTP',
+  'ESTJ',
+];
+interface Props {
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  disabled?: boolean;
+  limit: number;
+  checkedList: string[];
+}
+
+function StudyCreateMbtiSelect({
+  onChange,
+  disabled,
+  limit,
+  checkedList,
+}: Props) {
+  return (
+    <MbtisWrapper>
+      {mbtisList.map((mbti) => (
+        <Checkbox
+          onChange={onChange}
+          key={mbti}
+          value={mbti}
+          icon={
+            <MbtiTag
+              key={mbti}
+              mbti={mbti}
+              disabled={(limit > 2 ? true : false) || disabled}
+            />
```

Comment:
p3;

Button을 선택했을 때 뒤에 Ripple이 안보이는 것이 좋아보였습니다. :) 
`disableRipple`을 추가해서 봐보시면 어떠신가요? 

```suggestion
        <Checkbox
          onChange={onChange}
          key={mbti}
          value={mbti}
          icon={
            <MbtiTag
              key={mbti}
              mbti={mbti}
              disabled={(limit > 2 ? true : false) || disabled}
              disableRipple
            />
```